File: src/main/java/com/example/onemoretimejava/basic/EvenOrOdd.java
package com.example.onemoretimejava.basic;

public class EvenOrOdd {
    public static void main(String[] args) {
        evenOrOdd(0);
    }

    static void evenOrOdd(int n) {
        if (n % 2 == 0) {
            System.out.println("number is even");
        } else {
            System.out.println("number is odd");
        }
    }
}

File: src/main/java/com/example/onemoretimekotlin/basic/EvenOrOdd.kt
package com.example.onemoretimekotlin.basic


  /*  fun main() {
        evenOrOdd(15)
    }*/
  fun main(args: Array<String>) {
      println("Hello, world!")
      evenOrOdd(15)
  }

    fun evenOrOdd(n: Int) {
        if (n % 2 == 0) {
            println("number is even")
        } else {
            println("number is odd")
        }
    }

